src/java/picard/analysis/CollectRnaSeqMetrics.java

```diff
@@ -49,12 +49,48 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = "Collect metrics about the alignment of RNA to various functional classes of loci in the genome:" +
-                "coding, intronic, UTR, intergenic, ribosomal. Also determines strand-specificity for strand-specific libraries.",
-        usageShort = "Produces RNA alignment metrics for a SAM or BAM file",
+        usage = CollectRnaSeqMetrics.USAGE_SUMMARY + CollectRnaSeqMetrics.USAGE_DETAILS,
+        usageShort = CollectRnaSeqMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectRnaSeqMetrics extends SinglePassSamProgram {
+
+    static final String USAGE_SUMMARY = "Produces RNA alignment metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "<p>This tool takes a SAM/BAM file containing the aligned reads from an RNA-seq experiment and produces metrics describing the distribution " +
+            "of the bases within the transcripts.  It calculates the total numbers and the percentages of nucleotides within specific genomic regions" +
+            " including untranslated regions (UTRs), introns, intergenic sequences (between discrete genes), and peptide-coding sequences (exons).  " +
+            "This tool also determines the numbers of bases that pass the sequencing vendor's quality filters (PF_BASES). " +
+            "To learn more about the PF metric, please see the corresponding GATK " +
+            "<a href=\"https://www.broadinstitute.org/gatk/guide/article?id=6329\">Dictionary</a> entry.</p>" +
+            "" +
+            "<p>Other metrics include the median coverage (depth), 5'/3'-biases, and the numbers of reads with the correct/incorrect " +
+            "strand designation. The 5'/3'-bias results from errors introduced by reverse transcriptase enzymes during library construction, ultimately leading " +
+            "to the over representation of either the 5' or 3' ends of transcripts.  Please see the following " +
+            "<a href=\"https://www.biostars.org/p/102812/#102815\">link</a> for information on how this ratio is calculated. </p>" +
```

Comment:
Please don't link to external resources to explain Picard's calculations, that's kind of embarrassing. You can summarize how the calculation is done in one line, if it isn't already explained in the metrics definitions.